exercises/nucleotide-count/tests/Tests.elm

```diff
@@ -10,16 +10,19 @@ tests =
     describe "NucleotideCount"
         [ test "the solution is for the correct version of the test" <|
             \() -> Expect.equal 2 version
```

Comment:
Sometimes we change a test in a way that isn't backwards compatible. The version declaration in the exercise stub helps reviewers know what tests they were writing against. That's the idea at least... we haven't been perfectly consistent.